public/app/features/panel/metrics_panel_ctrl.ts

```diff
@@ -248,23 +248,27 @@ class MetricsPanelCtrl extends PanelCtrl {
     }
   }
 
-  getAdditionalMenuItems() {
+  async getAdditionalMenuItems() {
     const items = [];
     if (this.contextSrv.hasAccessToExplore() && this.datasource) {
       items.push({
         text: 'Explore',
-        click: 'ctrl.explore();',
+        click: 'ctrl.explore($event);',
```

Comment:
as mentioned below you do not need the click if you have a href

Reply:
👍